ccdb5_ui/templates/ccdb5_ui/ccdb-main.html

```diff
@@ -48,15 +49,9 @@
 
 {% block page_js %}
   <!--[if IE 9]><!-->
-  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=Intl.~locale.en"></script>
+  <script src="https://cdn.polyfill.io/v2/polyfill.js?features=default,Intl.~locale.en"></script>
```

Comment:
Not sure about loading polyfills off a CDN.

Reply:
We have [this PR](https://github.com/cfpb/ccdb5-ui/pull/105) where we are changing the polyfill call to happen only when the browser lacks the capabilities we need